website/docs/src/content/cloud/local-development.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Local development
+description: Run GenSX workflows locally with the dev server
+---
+
+# Local development server
+
+GenSX provides local development experience that mirrors the cloud environment, making it easy to build and test workflows on your machine before deploying them.
+
+## Starting the dev server
+
+The `gensx start` command launches a local development server with hot-reloading:
+
+```bash
+gensx start ./src/workflows.tsx
+🔍 Starting GenSX Dev Server...
+ℹ Using project name from gensx.yaml: support-tools
+ℹ Starting development server...
+✔ Compilation completed
+✔ Generating schema
+Importing compiled JavaScript file: /Users/evan/code/gensx-console/samples/support-tools/dist/src/workflows.js
+
+
+🚀 GenSX Dev Server running at http://localhost:1337
+🧪 Swagger UI available at http://localhost:1337/swagger-ui
+
+📋 Available workflows:
+- RAGWorkflow: http://localhost:1337/org/gensx/projects/support-tools/workflows/RAGWorkflow
+- AnalyzeDiscordWorkflow: http://localhost:1337/org/gensx/projects/support-tools/workflows/AnalyzeDiscordWorkflow
+- TextToSQLWorkflow: http://localhost:1337/org/gensx/projects/support-tools/workflows/TextToSQLWorkflow
+- ChatAgent: http://localhost:1337/org/gensx/projects/support-tools/workflows/ChatAgent
+
+✅ Server is running. Press Ctrl+C to stop.
+```
+
+## Development server features
+
+### Identical API shape
+
+The local API endpoints match exactly what you'll get in production, making it easy to test your workflows before deploying them:
+
+```
+http://localhost:1337/local/{project}/default/workflows/{workflow}/run
```

Comment:
This url is incorrect. Although my current PR shortens this even more than the suggestion below

```suggestion
http://localhost:1337/org/{org}/project/{project}/workflows/{workflow}
```

Reply:
i'd recommend just taking the suggestion and i'll go make updates with my PR